xee/README.md

```diff
@@ -51,6 +52,26 @@ Pretty-print `foo.xml`, result to stdout:
 xee indent foo.xml
 ```
 
+### Transform an XML document with XSLT
+
+Transform an XML document using an XSLT stylesheet:
+
+```
+xee transform stylesheet.xsl input.xml
```

Comment:
The command is `transform` here but I'd like it to be `xslt` analogous to `xpath` which is another CLI command.

https://github.com/Paligo/xee/issues/89